### DISCUSSION TITLE
Remote Control

### DISCUSSION THREAD

**Discussion by 1ay1** [igniterealtime/openfire-pade-plugin#347] [Ideas]
Along with the cursor sharing and screen sharing, how feasible is achieving the remote desktop control in the current system?

**Comment by deleolajide** [comment]
It was implemented in the older version of the chrome extension version of Pade using the remote desktop control feature of the Jitsi Meet desktop version. Unfortunately, remote desktop control is considered high security risk for hacking and the feature disabled. I have no plans to add it now or anytime soon.

It is a better to install a dedicated native application like TeamViewer than to allows direct remote desktop control from a web browser. It is too risky
